src/Model/FieldType/DateField.ts

```diff
@@ -7,8 +7,8 @@ export const DateField: IFieldType<Date> = {
   defaultValidationRules: { datetime: { dateOnly: true } },
   defaultValue: null,
   isValidType: (value) => value == null || isDate(value),
-  transform: (date: Date) => date != null ? format(date, "YYYY-MM-DD") : null,
-  normalize: (value: string | any): Date => (
+  transform: async (date: Date) => date != null ? format(date, "YYYY-MM-DD") : null,
```

Comment:
Why doesn't the `format` call require a promise?  Am I not thinking about it correctly since it's just a string?

Reply:
`format` is imported from `date-fns` and does not return a promise itself, so we do not need to `await`. Still, by adding the `async` keyword onto the closure, its output is a Promise (that resolves immediately). This is necessary since the signature of the `transform` method in the `IFieldType` interface was updated such that the method will always return a Promise for consistency.

Reply:
Okay.  That is what I was assuming and that makes total sense.

Reply:
👍